README.md

```diff
@@ -5,15 +5,15 @@
 This project is a game that has been developed in order to help teach kids python.
 The students need to code the behavior of the car to achieve the best score.
 
-Here is video of a race (running code from students):
+Here is video of a race, using drivers coded by students:
 
 <a href="http://www.youtube.com/watch?feature=player_embedded&v=BEV-CcqTOnw
 " target="_blank"><img src="rose-video-preview.jpg"
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas
+calculate the best path to take to avoid the pitfalls and collect bonus points.
```

Comment:
I don't think this improves the text, we are discussing here how a race car should be implemented.